hash/spectrum_cass.xml

```diff
@@ -6176,16 +6174,29 @@ license:CC0-1.0
 	</software>
 
 	<software name="scia" cloneof="sci">
-		<description>Special Criminal Investigation - Chase HQ 2 (Erbe)</description>
+		<description>Chase H.Q. II: Special Criminal Investigation (Erbe)</description>
 		<year>1990</year>
 		<publisher>Ocean</publisher>
+		<info name="region" value="Spain" />
 		<part name="cass" interface="spectrum_cass">
 			<dataarea name="cass" size="120141">
 				<rom name="chase hq 2 (erbe).tzx" size="120141" crc="73a875d3" sha1="19bdc80282fdcf9f386612b7830ac0c09e6999a4"/>
 			</dataarea>
 		</part>
 	</software>
 
+	<software name="scib" cloneof="sci">
+		<description>Chase H.Q. II: Special Criminal Investigation (Erbe, small cardboard case)</description>
+		<year>1990</year>
+		<publisher>Ocean</publisher>
+		<info name="region" value="Spain" />
```

Comment:
I could well be missing something, but weren’t Erbe the Spanish publisher for Ocean’s (and Gremlin’s) games?  Or are you consider Erbe to just be a regional distributor for games published by other companies?

Reply:
You're right, my mistake.
I corrected this one.